content/courses/intro-to-spark/chapters/05_Proof_Of_Functional_Correctness.rst

```diff
@@ -644,10 +649,19 @@ property.
 Ghost Procedures
 ~~~~~~~~~~~~~~~~
 
+.. todo::
+    
+    This section still seems very abstract. Can a more concrete example be developed?
+
 Ghost procedures can't affect the value of normal variables, so they're
 mostly used to perform operations on ghost variables or to group together a
 set of intermediate assertions.
 
+.. todo::
+
+    Hard to understand what this first sentence is trying to express;
```

Comment:
maybe replace "treatment" by "use"

Reply:
What do you think of "Organizing operations and assertions performed on ghost variables into ghost procedures has several advantages"

Reply:
looks good!